devdox/tests/unit_test/app/services/test_api_keys_service.py

```
@@ -0,0 +1,128 @@
+)
+from app.services.git_tokens_service import mask_token
+from app.utils.auth import UserClaims
+from app.exceptions.custom_exceptions import BadRequest
+from app.exceptions.exception_constants import (
+    UNIQUE_API_KEY_GENERATION_FAILED,
+    FAILED_GENERATE_API_KEY_RETRIES_LOG_MESSAGE,
+)
+from tests.unit_test.test_doubles.app.repository.api_key_repository_test_doubles import FakeApiKeyStore
+from tests.unit_test.test_doubles.app.service.api_keys_service_test_doubles import FakeAPIKeyManager
+
+@pytest.mark.asyncio
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)